### ISSUE TITLE
Inside `overflow: scroll` container => suggestions not overlapping the container

### ISSUE THREAD

**Issue by JaySunSyn** [ellipticaljs/paper-autocomplete#99]
How to use `paper-autocomplete` inside a scrollable container?

e.g. inside `paper-dialog-scrollable` the suggestions are not overlapping the dialog:

![image](https://user-images.githubusercontent.com/5872432/33549596-580a18dc-d8eb-11e7-81bd-dee637b2b813.png)

Thanks.

**Comment by ndormont** [comment]
@JaySunSyn , what's the issue?
Is it because the suggestions are not overlapping the dialog?
I have auto-complete paperchips in a dialogs, works just fine - but I see where you might have an issue if the filter input is too low, then the suggestion would be hidden/not shown, because they would be loaded inside the scrollable div.

**Comment by JaySunSyn** [comment]
@ndormont yes, the issue is that the suggestions are not overlapping the dialog.

A `paper-dropdown-menu` for example does overlap the the scrollable dialog:

![image](https://user-images.githubusercontent.com/5872432/33567686-4f3ae752-d924-11e7-94d0-e177aeedb9ef.png)

`paper-autocomplete` in a scrollable dialog:

![image](https://user-images.githubusercontent.com/5872432/33549596-580a18dc-d8eb-11e7-81bd-dee637b2b813.png)

`paper-autocomplete` works just fine in a non-scrollable dialog:

![image](https://user-images.githubusercontent.com/5872432/33567485-ad6acc8a-d923-11e7-9430-fb90207cf4f0.png)

**Comment by ndormont** [comment]
I had the same issue with  a `overflow: hidden` div. I solved it by changing it to `overflow: visible` - but in your case, you have an issue because you want it to be scrollable. I wonder, does paper-dropdown-menu uses z-index for their suggestions!?

**Comment by JaySunSyn** [comment]
Yes, indeed, I need it to be scrollable.

`paper-dropdown-menu` uses `iron-dropdown` uses `position: fixed` and `z-index`

**Comment by JaySunSyn** [comment]
This would work I guess:

```
paper-autocomplete {
  --paper-autocomplete-suggestions-wrapper: {
    position: fixed;
    width: initial;
  }
}
```

**Comment by JaySunSyn** [comment]
Still think that `position: fixed` should be the default.

**Comment by jhuesos** [comment]
Introducing this change could be considered a breaking change, that's why
in the past I preferred to don't change it.

In the future, we could push a new breaking change version and this can be
considered.

Thanks for the comments by the way

**Comment by ainvaltin** [comment]
I guess this is kind of the same problem so I won't open new issue but comment here - even when the `paper-autocomplete` input is near the bottom edge of the browser the autocomplete list opens below it and is not completely visible (see the picture):

![acpopup](https://user-images.githubusercontent.com/9789293/34501512-ee4004a4-f017-11e7-8d09-6636541423e6.png)

when navigating with keyboard the selection goes "out of view" and is thus pretty unusable.

The `paper-dropdown-menu` seems to manage always to figure out good height for the popup, so that it is always completely visible. Then again, it uses scrollbars in the popup even when there would be enough room to show all the items...

[Vaadin ComboBox](https://github.com/vaadin/vaadin-combo-box) has a nice feature that when it is near the bottom edge of the browser it opens it popup list above the input, not below it.

So perhaps borrowing some ideas from these two would help to make a perfect popup list :)

**Comment by bstmedia** [comment]
> This would work I guess:
> 
> ```
> paper-autocomplete {
>   --paper-autocomplete-suggestions-wrapper: {
>     position: fixed;
>     width: initial;
>   }
> }
> ```

It worked for me.  I would suggest to close this issue
